setup.py

```diff
@@ -1,46 +1,45 @@
+# -*- coding: utf-8 -*-
 """Setup file for chime
 """
 __version__ = "1.1.2"
 __author__ = "Predictive Healthcare @ Penn Medicine"
 
 from os import path
-from setuptools import setup, find_packages, find_namespace_packages
 
+from setuptools import setup, find_namespace_packages
+
+CWD = path.abspath(path.dirname(__file__))
+
+with open(path.join(CWD, "README.md"), encoding="utf-8") as inp:
```

Comment:
The setup.cfg file does this work for README.md

Reply:
setup.py is now from develop branch (just has selenium added to the list literal)